src/apps/cost_matrix_based_matching/cost_matrix_based_matching_lsh.cpp

```diff
@@ -87,15 +87,14 @@ int main(int argc, char *argv[]) {
   relocalizerPtr->train(featurePtrs);
 
   // initialize SuccessorManager
-  auto successorManagerPtr = SuccessorManager::Ptr(new SuccessorManager);
-  successorManagerPtr->setFanOut(parser.fanOut);
-  successorManagerPtr->setDatabase(database.get());
-  successorManagerPtr->setRelocalizer(relocalizerPtr);
+  std::unique_ptr<SuccessorManager> successorManager =
```

Comment:
nit: you can also use `auto` here 😉

Reply:
I see, thanks. Will keep that in mind :)